### DISCUSSION TITLE
CUDA Version: N/A

### DISCUSSION THREAD

**Discussion by mayday0208** [containerd/nerdctl#1022] [Q&A] [answered]
nerdctl run -it --rm --gpus capabilities=utility,device=GPU-3a23c669-1f69-c64e-cf85-44e9b07e7a2a nvidia/cuda:9.0-base nvidia-smi

when i use python pytorch,torch.cuda.is_available(),return false.  

how to use cuda with nerdctl run images?

**Comment by junnplus** [comment, marked as answer]
see https://github.com/containerd/nerdctl/issues/1023
